src/Tribe/Main.php

```diff
@@ -2148,6 +2154,20 @@ public static function get_series_start_date( $post_id ) {
 			}
 		}
 
+		/**
+		 * Runs the Event Scheduler to purge old events
+		 *
+		 * @return void
+		 */
+		public function runScheduler() {
```

Comment:
Dont use camelCase for our methods in PHP, we follow the WP Standards with snake_case